

mo: 40
Jo: [2.1417, 0.4167, 2.5417]

M: [  0.001, 0.001, 0.001,    0.00001, 0.00001, 0.00001  ]

a_Fp: 10 # 2
a_Fo: 10 # 2


# ==========  1: force_Dadapt   ==========
use_force: true
D_min: [ 30, 30, 30,    0.2, 0.2, 0.2 ]
D_max: [ 350, 350, 350,    10, 10, 10 ]


# # ==========  2: vel_Dadapt   ==========
# use_force: false
# D_min: [ 30, 30, 30,    0.2, 0.2, 0.2 ]
# D_max: [ 350, 350, 350,    10, 10, 10 ]


# # ==========  3: force_Dmin   ==========
# use_force: true
# D_min: [ 30, 30, 30,    0.2, 0.2, 0.2 ]
# D_max: [ 30, 30, 30,    0.2, 0.2, 0.2 ]

#
# # ==========  4: force_Dmax   ==========
# use_force: true
# D_min: [ 350, 350, 350,    10, 10, 10 ]
# D_max: [ 350, 350, 350,    10, 10, 10 ]
